app/routes/admin/settings/ticket-fees.js

```diff
@@ -5,5 +5,109 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Ticket Fees');
+  },
+  model() {
+    return [
```

Comment:
This isn't gonna be like this. 
It will be an array of objects like 
```js
[
  {
    currency: 'USD',
    serviceFee: 2.0
    maximumFee: 10.0
  }
]
```